paasta_tools/cli/utils.py

```diff
@@ -843,6 +847,36 @@ def get_instance_config(
     )
 
 
+def get_namespace_for_secret(
```

Comment:
thinking about this a little more: thoughts on renaming this to reflect that a collection will be returned?
```suggestion
def get_namespaces_for_secret(
```